test-clients/src/main/resource/contract/contracts/Transferring/Transferring.sol

```diff
@@ -1,21 +1,22 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.8.7;
```

Comment:
Why did you need to change the solidity code here? Does it mean the removal of the EthereumJ dependency will not allow us to test with older versions of Solidity later on?

Reply:
We can try to run the tests with the version that is on master (with 0.5.0 solidity version).